bower.json

```diff
@@ -19,14 +19,14 @@
     "/demo/"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#v0.9.0-rc.1",
-    "hydrolysis": "PolymerLabs/hydrolysis#^1.1.2",
     "marked-element": "PolymerElements/marked-element#^0.9.0",
     "paper-styles": "PolymerElements/paper-styles#^0.9.0",
+    "polymer": "Polymer/polymer#^0.9.0",
     "prism-element": "PolymerElements/prism-element#^0.9.0"
   },
   "devDependencies": {
     "test-fixture": "PolymerElements/test-fixture#^0.9.0",
+    "iron-component-page": "PolymerElements/iron-component-page#^0.9.0",
     "webcomponentsjs": "webcomponents/webcomponentsjs#^0.6.0",
```

Comment:
You need to update webcomponentsjs:
 "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"